security-scan.sh

```diff
@@ -18,7 +18,7 @@ done
 if [[ $RUN_BRAKEMAN = true ]]; then
   mkdir -p brakeman/reports
   chmod -R 0777 brakeman/reports
-  docker run -v "$(pwd):/tmp/" --rm -w /tmp/ codeclimate/codeclimate-brakeman brakeman -o brakeman/reports/brakeman-output.html
+  docker run -v "$(pwd):/tmp/" --rm -w /tmp/ codeclimate/codeclimate-brakeman:b804 brakeman -o brakeman/reports/brakeman-output.html
```

Comment:
Thank you! I was just about to add this :)

Reply:
No worries. I think we need to start building our own utility containers for stuff like this....

Reply:
I don't think it's worth the effort for one-tool containers like that. I do think, however, that we ought to start pinning known good versions. Even better, actually review the images to make sure there are no backdoors or surprises of some sort, and then pin by specific SHA that we reviewed.